src/appshell/internal/appshellconfiguration.h

```diff
@@ -106,6 +109,8 @@ class AppShellConfiguration : public IAppShellConfiguration, public muse::Inject
     muse::Ret writeSessionState(const QByteArray& data);
 
     muse::io::paths_t parseSessionProjectsPaths(const QByteArray& json) const;
+
+    QString m_preferencesDialogCurrentPageIdQString = "";
```

Comment:
Sorry, I missed one: 
```suggestion
    QString m_preferencesDialogCurrentPageId;
```
- the QString suffix is not necessary
- QString is a smart type in the sense that it initialises itself to an empty QString, so no need to assign a default value of `""`. In fact, if you assign `""`, then it will read the `""`, convert that to an UTF-16 string (which isn't very costly since it's empty anyway, but still) and then store that UTF-16 string. If you don't assign an explicit value, it will just create an empty UTF-16 string and that's all.